packages/vite/migrations.json

```diff
@@ -75,6 +75,10 @@
         "@vitest/coverage-istanbul": {
           "version": "^0.32.0",
           "alwaysAddToPackageJson": false
+        },
+        "@vitest/coverage-v8": {
+          "version": "^0.32.0",
+          "alwaysAddToPackageJson": false
```

Comment:
no change to the migration file is needed here. these are for past versions of nx.

Reply:
I've removed all changes to this file (assuming that's what you meant?)